unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Clipboard/Implementation/Web/Plugins/ClipboardWebGL.jslib

```diff
@@ -0,0 +1,41 @@
+var ClipboardWebGL = {
```

Comment:
That's true, but this is not something that needs maintaining or debugging, it's mainly hooking to callbacks from the browser

Reply:
1) All code needs maintaining and debugging down the road.

2) This code is interacting with `unityInterface` (kernel specific object) and relying on `eval` because we can't use promises here. This looks OK to you? You'd allow other devs to take this approach down the line?